trust-store-trust-policy-specification.md

```diff
@@ -1,154 +1,88 @@
-                    "-----BEGIN CERTIFICATE-----\ncertificate2\n----END CERTIFICATE-----\n"
-                ],
-                "tsaX509Certs": [
-                    "-----BEGIN CERTIFICATE-----\ntsaCertificate1\n-----END CERTIFICATE-----\n",
-                    "-----BEGIN CERTIFICATE-----\ntsaCertificate2\n-----END CERTIFICATE-----\n"
-                ]
-            }
-        },
-        "trust-store-name-2": {
-            "identities": {
-                "x509Certs": [
-                    "-----BEGIN CERTIFICATE-----\ncertificate1\n-----END CERTIFICATE-----\n",
-                    "-----BEGIN CERTIFICATE-----\ncertificate2\n----END CERTIFICATE-----\n"
-                ],
-                "tsaX509Certs": [
-                    "-----BEGIN CERTIFICATE-----\ntsaCertificate2\n-----END CERTIFICATE-----\n",
-                    "-----BEGIN CERTIFICATE-----\ntsaCertificate3\n----END CERTIFICATE-----\n"
-                ]
-            }
-        }
-    }
-}
+Contains a set of trusted identities through which trust is derived for the rest of the system. For X509 PKI, the trust store typically contains root certificates.
+
+- The Notary v2 trust store consists of multiple named collections of certificates, called named stores.
+- The trust store is a directory location, under which each sub directory is considered a named store, that contains zero or more certificates. The name of this sub directory is used to reference the specfic store in trust policy.
+- Certificates in a trust store are typically root certificates. Intermediate certificates can be used, but not recommended as intermediate certificates can be rotated more often and can break signature verification.
+
+Notary v2 uses following directory structure to represent the trust store. The example shows named stores `acme-rockets` and `wabbit-networks`, which are subseqently references in the trust policy. Without this reference, presence of a named store and certificates in it does not confer trust automatically to the named store. The trust store is configured ahead of verification time, by an out of band mechanism that is beyond the scope of this document. Different entities and organizations have their own processes and policies to configure and distribute trust stores.
+
+```text
+~/.notation/trust_store
+    /x509
+        /ca_roots
+            /acme-rockets
+                cert1.pem
+                cert2.pem
+                  /sub-dir       # ignored
+                    cert-3.pem   # ignored
+            /wabbit-networks
+                cert3.pem
+        /tsa_roots
+            /publicly-trusted-tsa
+                tsa-cert1.pem
 ```
 
-### Trust Store Properties
+The Trust store currently supports two kinds of identities, additional identities may be supported in future :
+
+- **Certificates**: The `x509/ca` trust store contains named stores that contain Certificate Authority (CA) root and intermediate certificates.
```

Comment:
Intermediate certificates should not be in the `roots` folders since they are not root CA certs.